point_pattern.py

```diff
@@ -109,7 +115,12 @@ def check_coincident(a, b):
     equal : bool
             Whether the points are equal
     """
-    return None  # Add the logic to check if coincident here
+    #return None # Add the logic to check if coincident here
+
+    if a[0]==b[0] and a[1]==b[1]:
+        coincident = True
+        return coincident
+
```

Comment:
Super close here.  Nice job!  Right now, if coincident is True, this returns `True` (exactly as it should).  What about if coincident is False?  Right now, this does not return anything, e.g. None.  We will talk a bit more about `if` statments later, so here is a way to do it without an if:

```python
return a[0]==b[0] and a[1]==b[1]
```

Alternatively, if you want to use an if statement, you could do:

```python

coincident = False
if a[0]==b[0] and a[1]==b[1]:
    coincident = True
    
return coincident
```